session.go

```diff
@@ -446,6 +452,9 @@ func (s *Session) recvLoop() error {
 		if err := handler(hdr); err != nil {
 			return err
 		}
+
+		// Ping is not required if data is flowing
+		atomic.StoreInt32(&s.needPing, 1)
```

Comment:
I don't see it being set back to 0 ever?
What if I had a lot of data flowing but then the stream goes silent?